Resources/Prototypes/Recipes/Lathes/botany.yml

```diff
@@ -37,3 +37,8 @@
   parent: BaseHydroToolRecipe
   id: HydroponicsToolClippers
   result: HydroponicsToolClippers
+
+- type: latheRecipe
```

Comment:
Should be in its own file in _Den/ but if you want to be lazy you can just fence it with den comments.